server/service/src/print/label.rs

```diff
@@ -91,7 +110,7 @@ pub fn print_prescription_label(
                 ^A0,25
                 ^FO,10
                 ^FB575,3,0,C
-                ^FD{item_details}\&^FS
+                ^FD{item_details}^FS
```

Comment:
Removed these new lines as didn't seem needed, maybe it was an attempt to avoid issues like we had.
@Chris-Petty ?